SharePic/app/src/main/java/edu/sdsu/cs/sharepic/activity/SettingsActivity.java

```diff
@@ -2,83 +2,68 @@
 
 import android.content.Intent;
 import android.os.Bundle;
-import android.support.v7.app.ActionBarActivity;
+import android.support.v7.app.AppCompatActivity;
 import android.view.Menu;
 import android.view.MenuItem;
+import android.view.ViewGroup;
 import android.widget.CompoundButton;
+import android.widget.LinearLayout;
 import android.widget.Switch;
+
+import java.util.ArrayList;
+
 import edu.sdsu.cs.sharepic.R;
-import edu.sdsu.cs.sharepic.model.Dropbox;
-import edu.sdsu.cs.sharepic.model.FlickrAccount;
+import edu.sdsu.cs.sharepic.model.Account;
+import edu.sdsu.cs.sharepic.model.LoginListener;
 
-public class SettingsActivity extends ActionBarActivity {
+public class SettingsActivity extends AppCompatActivity implements CompoundButton.OnCheckedChangeListener, LoginListener{
```

Comment:
Can u try and run the app on emulator. It was crashing when I used AppCompatActivity